website/docusaurus.config.js

```diff
@@ -7,7 +7,7 @@ const { absoluteUrl } = config;
 
 /** @type {Partial<import('@docusaurus/types').DocusaurusConfig>} */
 module.exports = {
-    title: 'Apify Documentation',
+    title: 'Apify Client for Python',
```

Comment:
this is the title of the whole docs.apify.com, I think we want it to be the same for all the projects - all the "child docusaurus projects" has it that way on purpose.

unless we want to have the title suffix different, which sounds a bit weird but it might work for our use case

Reply:
Yeah I wanted to discuss that thing - I think having the project name in the title makes more sense than just "Apify Documentation". Perhaps we could have both there, but it doesn't really make sense to me.

With just "Apify Documentation" there, there's no way to determine from the title which project you're in, because you have the title like "Quick Start | Apify Documentation" for both the client and the SDK, and it gets confusing.

BTW if you open https://docs.apify.com, the title is "Apify Documentation · Apify Documentation | Apify Documentation" 😄

Reply:
Yeah, I know, it is [already reported](https://github.com/apify/apify-docs/issues/519), but you know, there were more serious things to fix :D

Agreed we should have the project-level info in the title, I just want this to be consistent/coordinated.